src/test/unit_tests/jest_tests/test_nc_nsfs_schema_validation.test.js

```diff
@@ -1258,3 +1258,93 @@ describe('schema validation NC NSFS bucket', () => {
     });
 
 });
+
+describe('schema validation NC NSFS config schema', () => {
```

Comment:
@romayalon 
I will separate the files if you want to separate them for config schema in this PR (or I'll do it in my PR).

Reply:
I think that separating the tests by different "describe" sections is enough for now.
but I noticed that all tests codes are pretty repetitive( expect(err).toBeInstanceOf(RpcError); for example repeats 50 times), I think we can add helper functions instead that will make the test file a lot shorter, for example:
```
function create_account_object(),
function create_bucket_object(),
function assert_validation(schema_type, object_to_validate, fail_options = { message: '') })
```

Reply:
I wrote it this way so it will be super easy to investigate in case a test fails.
Anyway, we can take this discussion on my open PR.